test/common/libraries/os_properties.rb

```diff
@@ -19,6 +19,14 @@ def redhat_ubi?
     virtualized? && inspec.os.name == 'redhat'
   end
 
+  def redhat?
```

Comment:
suggestion, non blocking: do you really need those? I think you can use `os.redhat?` and `os.centos?`

Reply:
`os.redhat` includes RedHat, centos and other OSes. The alternative is to use `os[:name] == 'redhat'` or `os.name == 'redhat'` directly